backend/coins/btc/transactions/transactions_test.go

```diff
@@ -190,6 +191,39 @@ func (s *transactionsSuite) TestUpdateAddressHistorySingleTxReceive() {
 	require.Equal(s.T(), expectedHeight, transactions[0].Height)
 }
 
+// TestTxSerialization checks that Tx marshaling/unmarshaling work with both current and legacy
```

Comment:
I think the point of this test was to document what kind of tx formats are stored in the tx database in the wild, and the original issue we had came from loading a tx with an old serialization from the DB. 

I think maybe `db.TxInfo` was the one that failed, as it loads by deserializing DBTxInfo? See  https://github.com/digitalbitbox/bitbox-wallet-app/blob/fb72430e49af1f71605587b723cb826089500ed4/backend/coins/btc/transactions/db.go#L28

As such, I think the test should be in db_test.go, testing using DBTxInfo instances. WDYT?

Reply:
You're right! But I guess you meant transactionsdb_test.go. Moved there :+1: